openedx/features/subscriptions/templates/subscriptions/edx_ace/common/base_body.html

```diff
@@ -0,0 +1,204 @@
+{% load i18n %}
```

Comment:
Can you share the path from where you pulled this template?

Reply:
https://github.com/edly-io/edly-edx-themes/blob/develop-juniper/st-lutherx/lms/templates/ace_common/edx_ace/common/base_body.html

Reply:
What if we achieve this via JS? 
@zubair-arbi What do you say?

Reply:
Yes, we might be able to achieve this via JS, but every time we make any changes for other emails we will have to make those changes via JS over here (adding some extra effort), whereas in this case we would simply need to replicate any changes made in `base_body.html` template over here as well.
@zubair-arbi which approach should we go for?

Reply:
Right.